features/networking/pod.feature

```diff
@@ -190,3 +190,128 @@ Feature: Pod related networking scenarios
     Then the step should succeed
     And the output should not contain "ingress_policing_rate: 1953"
 
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-23890
+  @admin
+  Scenario: A pod with or without hostnetwork cannot access the MCS port 22623 or 22624 on the master
+    Given evaluation of `env.master_hosts.first.local_ip` is stored in the :master_ip clipboard
+    Given I select a random node's host
+    Given I have a project
+    #pod-for-ping will be a non-hostnetwork pod
+    And SCC "privileged" is added to the "system:serviceaccounts:<%= project.name %>" group
+    And I have a pod-for-ping in the project
+
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22623/config/master | -k |
+    Then the output should contain "Connection refused"
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22624/config/master | -k |
+    Then the output should contain "Connection refused"
+    
+    #hostnetwork-pod will be a hostnetwork pod
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/anuragthehatter/v3-testfiles/master/networking/hostnetwork-pod.json |
+      | n | <%= project.name %>                                                                                   |
+    Then the pod named "hostnetwork-pod" becomes ready
+    #Pods should not access the MCS port 22623 or 22624 on the master
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22623/config/master | -k |
+    Then the output should contain "Connection refused"
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22624/config/master | -k |
+    Then the output should contain "Connection refused"
+
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-23891
+  @admin
+  Scenario: A pod cannot access the MCS port 22623 or 22624 via the SDN/tun0 address of the master
+    Given I use the first master host		
+    And I run commands on the host:
+      | ifconfig tun0 \| grep -w inet \| awk '{print $2}' |
+    Then the step should succeed
+    And evaluation of `@result[:response].strip` is stored in the :master_tun0_ip clipboard
+    
+    Given I select a random node's host
+    And I have a project
+    #pod-for-ping will be a non-hostnetwork pod
+    And I have a pod-for-ping in the project
+    #Curl on Master's tun0 IP to make sure connections are blocked to MCS via tun0
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_tun0_ip %>:22623/config/master | -k |
+    Then the output should contain "Connection refused"
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_tun0_ip %>:22624/config/master | -k |
+    Then the output should contain "Connection refused"
+
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-23892
+  @admin
+  Scenario: A pod cannot access the MCS via an egress router in http proxy mode
```

Comment:
@anuragthehatter  please help update the case according to the comment int the jira card. 
1. update the case only can be tested on bare-metal/vsphere
2. using the --proxy $egress-routerip:port for line 260 and 263.

Reply:
Yea but it seems like this test case needs to be automated later when bare-metal or vpshere is frequently available to try testing before putting this particular scenario in master. If we agree, we can merge this PR as nothing more needs to be done else we need to wait for bare-metal/vsphere to become available (right now its not). I have removed this scenario for now as i don't want to push changes without testing locally. LMK